src/transformers/visitors/checks/interactsWithSecretVisitor.ts

```diff
@@ -50,6 +74,14 @@ export default {
     },
   },
 
+  FunctionDefinition: {
+    exit(path: NodePath) {
+      path.traversePathsFast(interactwithInferSecret, {
```

Comment:
I think this should be inferInteractsWithSecret, as we are inferring whether an identifier interacts with secret.